pkg/client/didexchange/client_test.go

```diff
@@ -221,5 +223,30 @@ type mockDIDCreator struct {
 }
 
 func (m *mockDIDCreator) CreateDID(opts ...wallet.DocOpts) (*did.Doc, error) {
-	return &did.Doc{}, nil
+	const didContext = "https://w3id.org/did/v1"
```

Comment:
May be for next patch, use this
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/internal/mock/common/did/mock_creator.go